package.json

```diff
@@ -56,7 +56,7 @@
     "format": "prettier --check \"**/*.{json,md,yml,yaml}\"",
     "format:fix": "prettier --write \"**/*.{json,md,yml,yaml}\"",
     "husky:lint-staged": "lint-staged",
-    "husky:pre-commit": "lint-staged && npm run spellcheck",
+    "husky:pre-commit": "lint-staged && npm run spellcheck && exec < /dev/tty && bin/otomi bootstrap",
```

Comment:
I am now looking at this code after commenting in the PR thread, but see this is not intuitive, and depends on $CLOUD & $CLUSTER being set in env.

So this will fail when that is not the case.

Reply:
That makes sense. Thanks for the review anyway.